charts/dvpe-deployment-gloo/values.yaml

```diff
@@ -194,38 +194,68 @@ gloo:
           cookie_domain:
-        backendPlugin:
-          # gloo.authConfig.spec.configs.backendPlugin.enabled -- If `enabled` set to true the backend plugin will be used
+          # gloo.authConfig.spec.configs.oauth.strong_authentication_level -- The strong authentication level. Possible values are: 4000, 7000. If not set, there is no strong authentication.
+          strong_authentication_level:
+        tokenValidationPlugin:
+          # gloo.authConfig.spec.configs.tokenValidationPlugin.enabled -- If `enabled` set to true the backend plugin will be used
           enabled: false
-          # gloo.authConfig.spec.configs.backendPlugin.name -- `Name` of the cache plugin
-          name: AuthFlowBackend
+          # gloo.authConfig.spec.configs.tokenValidationPlugin.name -- `Name` of the auth token validation plugin
+          name: AuthTokenValidation
           config:
-            # gloo.authConfig.spec.configs.backendPlugin.config.cacheTableName -- `cacheTableName` of the auth cache
-            cacheTableName: auth-cache-prod
-            # gloo.authConfig.spec.configs.backendPlugin.config.awsRegion -- `awsRegion` where the cache is located
-            awsRegion: eu-west-1
-            # gloo.authConfig.spec.configs.backendPlugin.config.oidcUrl -- `oidcUrl` where the access token can be verified at the IDP
+            cache:
+              # gloo.authConfig.spec.configs.tokenValidationPlugin.config.cache.enabled -- if `enabled` is false, no cache is used
+              enabled: false
+              # gloo.authConfig.spec.configs.tokenValidationPlugin.config.cache.tableName -- `tableName` of the auth cache
+              tableName: auth-cache-prod
+              # gloo.authConfig.spec.configs.tokenValidationPlugin.config.cache.awsRegion -- `awsRegion` where the cache is located
+              awsRegion: eu-west-1
+            # gloo.authConfig.spec.configs.tokenValidationPlugin.config.oidcUrl -- `oidcUrl` where the access token can be verified at the IDP
             oidcUrl:
-            # gloo.authConfig.spec.configs.backendPlugin.config.jwksUrl -- `jwksUrl` where the JWKS (JSON Web Key Store) can be retrieved from the IDP
-            jwksUrl:
-        m2mPlugin:
-          # gloo.authConfig.spec.configs.m2mPlugin.enabled -- If `enabled` set to true the machine to machine plugin will be used
+            # gloo.authConfig.spec.configs.tokenValidationPlugin.config.allowedClientIds -- `allowedClientIds` **list (NOT string!)** of ids that are allowed by the plugin. If not given at all, all clients are allowed. If [], then no client is allowed. If [a, b], then a, b are allowed
+            allowedClientIds:
+            # gloo.authConfig.spec.configs.tokenValidationPlugin.config.authLevel -- `authLevel` strong authentication level: nil/4000/7000
+            authLevel:
```

Comment:
We should use the same name as in the tokenValidationPlugin --> `strongAuthenticationLevel`.
Can you use the description from above as well?

Reply:
done